instrumentation/reactor-3.1/library/src/main/java/io/opentelemetry/instrumentation/reactor/ReactorTracing.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.reactor;
+
+import io.opentelemetry.context.Context;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class ReactorTracing {
```

Comment:
I think we'll eventually need to provide help for actually adding spans for `Mono`s since it's pretty tedious. Since different `Mono`s may have very different attributes / semantics, I guess it will be common to have multiple `ReactorTracing` in an application.

Mix-and-match of that and a static initializer like this one does seem like it can get confusing. How about we rename this to `ReactorContextPropagation` to make clear this is all about propagation?